packages/wells/src/client/api/wells.ts

```diff
@@ -40,10 +40,24 @@ export class Wells extends AssetsAPI {
         id: asset.id,
         name: asset.name,
         wellHeadLocation: Wells.mapToWellHeadLocation(asset),
+        dataSource: Wells.getLabel('DataSource', asset.labels),
```

Comment:
Maybe it would be worth having an anum somwhere with the possible values of this before it explodes (at some point we will also have to add the relationship types and maybe some other "constants" that may be better organized somewhere together)

Reply:
@drsphelps mabye something like [this](https://github.com/cognitedata/cognite-sdk-js/blob/master/packages/core/src/constants.ts)

Reply:
The only thing is that every time someone on the BP project decides to add a new label, we would have to come back here and extend the enum? Or do you just mean data source, and that data source is well defined?

Reply:
I guess we should avoid having constants/enums that are specific to either AkerBP or BP. I'm thinking just adding regular constants and well-defined terms like i.e 'DataSource'

Reply:
Ah I see what you mean now